Samples/src/Arithmetic/README.md

```diff
@@ -1,10 +1,18 @@
-﻿# Adder Example #
+﻿# Adder Sample #
 
-This sample shows how to add two quantum integers using the arithmetic functionality in the canon.
+This sample shows how to add two integers using the arithmetic functionality in the Q# canon.
+Two examples of adders implemented in this sample are that of [Takahashi *et al.*](https://arxiv.org/abs/0910.2530) and [Cuccaro *et al.*](https://arxiv.org/abs/quant-ph/0410184).
 
 ## Running the Sample ##
 
 To run this sample, you will need Jupyter Notebook and the IQ# kernel.
+For instructions on how to set up these tools see the instructions [here](https://docs.microsoft.com/en-us/quantum/install-guide/?view=qsharp-preview).
```

Comment:
By way of explanation for my suggestions on this line, it's generally best to leave languages out of docs URLs so that the link automatically redirects based on the user's language. For search engines, it's also helpful if the link text is more descriptive than "here."
```suggestion
For instructions on how to set up these tools see the [getting started guide](https://docs.microsoft.com/quantum/install-guide/).
```